ballerina/modules/wssec/document.bal

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/jballerina.java;
+
+class Document {
+    
+    private handle nativeDocumentBuilder;
```

Comment:
Since we are not accessing this in `ballerina` side. We could store this native data

Reply:
Fixed in [0739c63](https://github.com/ballerina-platform/module-ballerina-soap/pull/103/commits/0739c63e4f9e8df38cc1c78d7089ae21935ab3d1)